docs/sdk/authentication/session-sigs/intro.md

```diff
@@ -11,116 +11,63 @@ import FeedbackComponent from "@site/src/pages/feedback.md";
 `SessionSigs` are only available on Ethereum and are heavily in development, so things may change. Be sure to use the latest version of the Lit JS SDK.
```

Comment:
> `SessionSigs` are only available on Ethereum

I'm not exactly sure what this is trying to say, and I think it would confuse users since it sounds like you can only use Session Signatures if you're using Ethereum with Lit

Reply:
Removed this. Was included only because the current docs have it